ui/admin/tests/acceptance/storage-buckets/selectors.js

```diff
@@ -3,6 +3,7 @@
  * SPDX-License-Identifier: BUSL-1.1
  */
 
+export const STORAGE_BUCKET_TITLE = 'Storage Buckets';
```

Comment:
Since this selector is within the storage bucket specific selectors file, adding STORAGE_BUCKET_* to the selector name, seems redundant.
What are your thoughts on renaming this selector to TITLE, TITLE_SECTION or something like that?

Reply:
Great thought. I opted to include `STORAGE_BUCKET` in the title name because we are calling the import `selectors`. If we maybe changed the import name to `SBselectors` we could leave `STORAGE_BUCKET` out of the title name. This is something we can discuss during tech time maybe?